app/views/layouts/partials/_searchResults.html.erb

```diff
@@ -35,11 +35,15 @@
 
         <thead>
           <tr>
-            <th>Name</th>
-            <th>Address</th>
-            <th>Zipcode</th>
+
+
+            <th><%= button_to "Name", search_path, method: :get, params: { :county => params[:county], :item => params[:item], :zip => params[:zip], :sortby => "name" }%></th>
+            <th>
+              Address <%= button_to "by Zip Code", search_path, method: :get, params: { :county => params[:county], :item => params[:item], :zip => params[:zip], :sortby => "zipcode" }%>
+            </th>
+            <th>Website</th>
             <th>Phone</th>
-            <th>Validated</th>
+            <th><%= button_to "Verified", search_path, method: :get, params: { :county => params[:county], :item => params[:item], :zip => params[:zip], :sortby => "verified" }%></th>
```

Comment:
Also, we're going to need to add the ability to verify on this page. That's really important because it's the time they're most likely to verify a location and items.